src/Mod/Fem/femexamples/examplesgui.py

```diff
@@ -236,10 +236,10 @@ def run(self):
         # if done this way the Python commands are printed in Python console
         FreeCADGui.doCommand("from femexamples.manager import run_example")
         if solver is not None:
-            FreeCADGui.doCommand("run_example(\"{}\", solver=\"{}\")"
+            FreeCADGui.doCommand("run_example(\"{}\", solver=\"{}\", run_solver=True)"
                                  .format(str(example), str(solver)))
         else:
-            FreeCADGui.doCommand("run_example(\"{}\")".format(str(example)))
+            FreeCADGui.doCommand("run_example(\"{}\", run_solver=True)".format(str(example)))
```

Comment:
Just to confirm -- even though `solver` is None in this case, you still want to set `run_solver=True`?

Reply:
The difference is that in one case it uses the specified solver and in the other the default solver used in the analysis setup.
In the examples tree, if you run an analysis from "solvers" group, the first variant is used. On the other hand, if you select an analysis from some other group, the default solver is used.